public/antigravity.css

```
@@ -192,3 +192,1683 @@ body.antigravity-theme .ignition-view__inner {
+
+html[data-theme="dark"] body.antigravity-theme #ignition-view {
+  background-color: #18181b;
+  background-image:
+    radial-gradient(ellipse 70% 54% at 50% 26%, rgba(158, 139, 255, 0.12) 0%, transparent 68%),
+    radial-gradient(ellipse 42% 34% at 18% 78%, rgba(141, 134, 201, 0.09) 0%, transparent 60%),
+    radial-gradient(ellipse 38% 30% at 84% 74%, rgba(60, 221, 199, 0.07) 0%, transparent 56%);
+}
+
+/* ── Ignition title: larger, tighter, more expressive ──────── */
+body.antigravity-theme .ignition-title {
+  font-family: 'Outfit', 'Manrope', 'Inter', sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)